app/api/dao/admin.py

```diff
@@ -2,9 +2,24 @@
 
 
 class AdminDAO:
+    """ Data Access Object for Admin funcionalities.
```

Comment:
remove the space between `"` and `Data`; Also `funcionalities` to `functionalities`
From
```
""" Data Access Object for Admin funcionalities.
```
To
```
"""Data Access Object for Admin functionalities.
```

Reply:
@isabelcosta done.

Reply:
Now, I'm feeling picky, I told you about the typo, but I also mentioned about the extra space character you have between `"` and `Data`, The other doc string you have start with `"""Something` here you have a space, and I don't know why, is it on purpose? Can you fix that please? @sarnava1